nixos/modules/system/boot/loader/raspberrypi/uboot-builder.nix

```diff
@@ -1,13 +1,15 @@
-{ config, pkgs, configTxt }:
+{ pkgs, version, configTxt }:
 
 let
   cfg = config.boot.loader.raspberryPi;
```

Comment:
I dropped this line when applying to make it evaluate.

Reply:
Oops, sorry, I had fixed this on my production branch, but I forgot to apply it to the PR.